source/src/protocols/cyclic_peptide_predict/SimpleCycpepPredictApplication.cc

```diff
@@ -3286,6 +3439,79 @@ SimpleCycpepPredictApplication::add_closebond_logic_thioether_lariat(
 	}
 }
 
+//Lanthionine for now
+void
+SimpleCycpepPredictApplication::add_closebond_logic_lanthipeptide(
+	core::pose::PoseCOP pose,
+	core::Size const cyclization_point_start,
+	core::Size const cyclization_point_end,
+	protocols::generalized_kinematic_closure::GeneralizedKICOP genkic
+) const {
+	debug_assert( cyclization_type() == SCPA_lanthipeptide );
+	core::Size ala_res = 0;
+	core::Size cys_res = 0;
+	if ( pose->residue_type(cyclization_point_start).base_name() == "CYS" ) {
```

Comment:
Is the deliberate comparison to CYS intended, or is generlization to sidechain thiol a better choice here?

Reply:
For lanthipeptides, the mechanism of formation involves an L-cysteine residue.

Reply:
I'm just thinking about future engineering applications. Would there (could there) ever be the case where you might want to model a lanthipeptide with some non-canonical amino acid which isn't starting with an L-cysteine?